examples/browser-vue/package.json

```diff
@@ -48,6 +48,7 @@
     "eslint-plugin-vue": "^7.2.0",
     "playwright": "^1.12.3",
     "rimraf": "^3.0.2",
-    "test-util-ipfs-example": "^1.0.2"
+    "test-util-ipfs-example": "^1.0.2",
+    "util": "^0.12.4"
```

Comment:
`util` is not being used in the code. It is require for bundle purposes?

Reply:
It fixes this: https://github.com/ipfs-examples/js-ipfs-examples/runs/3619259311?check_suite_focus=true